src/main/java/com/poliscrypts/api/repository/CompanyRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.poliscrypts.api.repository;
+
+import com.poliscrypts.api.entity.Company;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+import java.util.Optional;
+
+@Repository
+@Transactional
```

Comment:
In the future there may be several writes, this allows them to be included in a single atomic operation, for optimization purposes only :)
by the way  it will be eliminated since the app context it is about simple crud operations